README.md

```diff
@@ -7,14 +7,13 @@ A fast parameter sampler and optimiser based on differential evolution.
 About
 --
 
-Diver is written in Fortran2003.  We originally wrote it with applications in particle physics and astronomy in mind, but it has since found broad applicability in other fields.
+Diver is written in Fortran2008 and uses MPI.  We originally wrote it with applications in particle physics and astronomy in mind, but it has since found broad applicability in other fields.
```

Comment:
It's because of the updated MPI module I introduced in b2d44d6bd56f3e1633f68e3d0955db18728d4c3b for [v1.0.8](https://github.com/patscott/Diver/releases/tag/v1.0.8).

Reply:
makes sense!